.github/workflows/snyk-sbom.yml

```diff
@@ -0,0 +1,25 @@
+name: Snyk - Software Bill of Materials (SBOM)
+on:
+  release:
+    types: [published]
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - name: Generate SBOM
+        uses: snyk/actions/node@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
You should have rights to introduce a new global mattermost org secret scoped to mattermost repo, for the SNYK_TOKEN.  🚀 
If not, let us know.

Reply:
@phoinixgrr i don't think I have permission